internal/ioutil/io.go

```diff
@@ -71,10 +49,10 @@ func CopyBuffer(dst io.Writer, src io.Reader, buf []byte, desc ocispec.Descripto
 		return errors.New("digest verification failed")
 	}
 
-	return ensureEOF(lr)
+	return EnsureEOF(lr)
 }
 
-func ensureEOF(r io.Reader) error {
```

Comment:
We need comment doc for every public func.

Reply:
Added the doc comment.